frontend/javascripts/oxalis/view/action_bar_view.tsx

```diff
@@ -198,16 +199,17 @@ class ActionBarView extends React.PureComponent<Props, State> {
     location.href = `${location.origin}/annotations/${annotation.typ}/${annotation.id}${location.hash}`;
   };
 
-  renderStartAIJobButton(): React.ReactNode {
+  renderStartAIJobButton(disabled: boolean): React.ReactNode {
+    const tooltipText = disabled ? "The dataset needs to have a color layer to start AI processing jobs." : "Start a processing job using AI" // TODO maybe hint how to create one? 
```

Comment:
I'd say the hint is not needed. It's the default case that datasets have a color layer, because that's what is recorded by the microscope. As you noted before, this check is very edge-casey so it is a safeguard to avoid crashes but doesn't need more explanation at this point.

On another note, it would be great if we find a more self-explanatory word for what a "color" layer is, but that's a different issue :sweat_smile:

Reply:
okay, thank you!